libs/build_dod_dataset.py

```diff
@@ -347,6 +352,7 @@ def get_usa_by_states_df():
     states_final = states_final.fillna(NULL_VALUE)
     states_final['Combined Key'] = states_final['Province/State']
     states_final['State/County FIPS Code'] = states_final['Province/State'].map(us_fips)
+    states_final['State/County FIPS Code'] = states_final['State/County FIPS Code'].astype(str).str.zfill(5)
```

Comment:
is this necessary if we have performed the padding earlier in this function at raw_df?

Reply:
yea, it's probably overkill, then again, most of this code is.